variantlib/plugins/_subprocess.py

```diff
@@ -46,19 +57,30 @@ def load_plugins(plugin_apis: list[str]) -> Generator[PluginType]:
         else:
             plugin_instance = plugin_callable
 
-        required_attributes = PluginType.__abstractmethods__
+        if isinstance(plugin_instance, PluginStaticType):
```

Comment:
`isinstance()` won't work here since the protocol allows optional methods (`get_build_setup`). Also, I'm not sure if it's going to work with `ModuleType` and so on.